### ISSUE TITLE
manage_podcast cap doesn't work without manage_options also

### ISSUE THREAD

**Issue by justintadlock** [CastosHQ/Seriously-Simple-Podcasting#650]
When creating a custom podcaster role and giving that role all podcast-related capabilities, including `manage_podcast`, users of this role cannot save settings.

Roles with `manage_podcast` can view the plugin settings screen.  However, when attempting to save any setting, the user is presented with the following error:

> You need a higher level of permission.
>
> Sorry, you are not allowed to manage options for this site.

The only way I've managed to make this work is for the role to also have the core `manage_options` capability, which is not desirable in this situation.

**Comment by TheCraigHewitt** [comment]
Hi @justintadlock thanks for bringing this to our attention. We'll work on getting this updated here shortly and ought to have this in an upcoming release in the next couple of weeks.  If we can do anything else to help out in the meantime please let us know.

**Comment by zahardev** [comment]
Hi @justintadlock! 
Please make sure you gave the podcaster role all the episode-related capabilities ( "edit_podcast", "read_podcast" etc. )
Here is the full list of capabilities:
```
'capabilities'        => array(
				'edit_post'              => 'edit_podcast',
				'read_post'              => 'read_podcast',
				'delete_post'            => 'delete_podcast',
				'edit_posts'             => 'edit_podcasts',
				'edit_others_posts'      => 'edit_others_podcasts',
				'publish_posts'          => 'publish_podcasts',
				'read_private_posts'     => 'read_private_podcasts',
				'delete_posts'           => 'delete_podcasts',
				'delete_private_posts'   => 'delete_private_podcasts',
				'delete_published_posts' => 'delete_published_podcasts',
				'delete_others_posts'    => 'delete_others_podcasts',
				'edit_private_posts'     => 'edit_private_podcasts',
				'edit_published_posts'   => 'edit_published_podcasts',
				'create_posts'           => 'create_podcasts',
			),
```

In the next plugin release, we're going to add the role "Podcast Editor", which gives the user permission to edit only episodes and media files.
So, if the above solution doesn't work for you, you could try that role instead.

**Comment by zahardev** [comment]
@justintadlock I could reproduce the issue you described, so it will be fixed in the next release. Thanks for your feedback!

**Comment by zahardev** [comment]
@justintadlock This issue is fixed within the 2.6.3 version. 
Additionally, we added custom roles for it: Podcast Editor (can manage only podcasts and media) and Podcast Manager (can manage podcasts, media, series, and podcast settings).
